lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -3958,7 +3948,7 @@ ADContext::declareExternalAssociatedFunction(
   auto *assocFn = fb.createFunction(
       SILLinkage::PublicExternal, name, assocFnTy,
       /*genericEnv*/ nullptr, originalLoc, original->isBare(), IsNotTransparent,
-      original->isSerialized(), original->isDynamicallyReplaceable());
+      IsNotSerialized, original->isDynamicallyReplaceable());
```

Comment:
Perhaps this change is not necessary?

Reply:
Reverted change in https://github.com/apple/swift/pull/22280/commits/6e8f9d503b12611a367527a41b1e98c0ef3ec9d2.